package.json

```diff
@@ -53,6 +53,7 @@
     "ipld-dag-cbor": "~0.11.1",
     "ipld-dag-pb": "~0.11.0",
     "ipld-eth-star": "^1.4.1",
+    "ipld-git": "file:../js-ipld-git",
```

Comment:
woa, this one is new to me. I never saw a `file:` in a package.json. This is almost like gx-rw

Reply:
Once the changes requested in ipld-git are done, let's update this line